src/query/service/src/pipelines/processors/transforms/window/transform_window.rs

```diff
@@ -707,6 +756,9 @@ where T: Number + ResultTypeOfUnary
             prev_frame_start: RowPtr::default(),
             prev_frame_end: RowPtr::default(),
             peer_group_start: RowPtr::default(),
+            peer_group_end: RowPtr::default(),
+            peer_group_ended: false,
+            need_peer: true,
```

Comment:
Not all functions need peer.

Reply:
Theoretically yes, but currently only `cume_dist` is needed, I'll modify it.